README.md

```diff
@@ -20,7 +20,9 @@ Once installed, you can use the Galactica Guardian CLI to perform various tasks.
 manage certificates, interact with the blockchain, and handle key operations. Each command supports the `--help` flag
 for detailed information and usage guidelines.
 
-### Commands Overview:
+The CLI **requires** the [Merkle Proof Service](https://github.com/Galactica-corp/merkle-proof-service) for issuing and revoking Zero Knowledge Certificates. Ensure that this service is running and accessible to perform these operations.
```

Comment:
Please, format this line as every other line in this document: break line at 120 characters limit